src/app/lib/analyticsUtils/index.test.js

```diff
@@ -429,39 +430,63 @@ describe('getAtiUrl', () => {
 describe('getClickInfo', () => {
   const params = {
     service: 'service',
-    component: 'component',
-    label: 'label',
+    componentName: 'component',
+    componentInfo: {
+      creationLabel: 'creationLabel',
+      url: 'url.com',
+      format: {
+        parent: 'container-component',
+        child: 'child',
+      },
+    },
     type: 'type',
   };
 
   it('should return url section', () => {
     expect(getClickInfo({}, params)).toEqual(
-      'PUB-[service-component]-[=type]-[label]-[PAR=container-component::name~CHD=brand-top]-[]-[]-[]-[/]',
+      'PUB-[service-component]-[=type]-[creationLabel]-[PAR=container-component::name~CHD=child]-[]-[]-[]-[url.com]',
```

Comment:
Is this in quite the right format? Looking right at the bottom of [this page](http://bbc.github.io/echo-docs/documentation/reporting-ati/#user-action-events) it looks a bit different

Reply:
Here is a comparison of the two:
<img width="653" alt="image" src="https://user-images.githubusercontent.com/43134742/69560844-4b43a900-0fa4-11ea-9c5a-dbbe960b6f4f.png">

I'm afraid I don't know if the way we're using it is different to those docs or not.

Reply:
this is one of the things that we will need to talk to Tanvee about. She's the one who knows the ins and outs of ATI. I was trying to match the values with the labels as shown in ATI Tag Inspector